wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -276,11 +276,17 @@ def test_page_is_public(context, page):
 @register.simple_tag
 def hook_output(hook_name):
     """
-    Example: {% hook_output 'insert_editor_css' %}
+    Example: {% hook_output 'insert_global_admin_css' %}
     Whenever we have a hook whose functions take no parameters and return a string, this tag can be used
     to output the concatenation of all of those return values onto the page.
     Note that the output is not escaped - it is the hook function's responsibility to escape unsafe content.
     """
+    if hook_name == "insert_editor_css":
+        warn(
+            "Querying `insert_editor_css` is deprecated, use `insert_global_admin_css` instead.",
```

Comment:
A bit of an issue here...
We're not expecting Wagtail site authors to use `{% hook_output 'insert_editor_css' %}` in their own code - this template tag is used within Wagtail's own templates (e.g. [admin_base.html](https://github.com/wagtail/wagtail/blob/aa9e9cddc7ff66e030a9b1ffc5eff6516851e56e/wagtail/admin/templates/wagtailadmin/admin_base.html#L8)) to output any tags that a developer sets up via `@hooks.register('insert_editor_css')`. The `{% hook_output 'insert_editor_css' %}` line will have to remain in Wagtail's own code (since we're not fully removing the code just yet), so putting the warning here will mean it'll get triggered every time that bit of the Wagtail base template runs, even if the site author isn't doing anything with it. What we really want to do here is raise the warning _only_ if the site author has made use of that hook - in other words, only if we find that the `snippets` list we build below is non-empty. The warning message can then be changed to something like: "The`insert_editor_css` hook is deprecated - use `insert_global_admin_css` instead.", since _querying_ the hook is the bit that Wagtail itself is doing.

Also, rather than raising `DeprecationWarning` - Wagtail defines a set of warning classes in [wagtail.utils.deprecation](https://github.com/wagtail/wagtail/blob/main/wagtail/utils/deprecation.py) - in this case, we'd expect this change to land in a 5.x release and the final removal to happen in 6.0, so `RemovedInWagtail60Warning` is the one to use.

Reply:
Ahh I see! Makes sense thanks! 
I changed to check also the `snippets` list!